### PR TITLE
Update TFS redirect and make redirect to Sonarsource Jira definitive

### PR DESCRIPTION
**Pull request by hgomez-sonarsource**
Ensure requests to SONARSCTFS goes in correct location
Make all requests to jira.sonarsource.com should be definitive (301)

### CODE REVIEW

File: includes/jira.codehaus.org.inc [hunk 1]
```diff
@@ -7,8 +7,9 @@
 ##################################
 # Owned by: Olivier Gaudin       #
 ##################################
-RewriteRule "^/browse/SONAR(.*)$" "http://jira.sonarsource.com/browse/SONAR$1" [L]
-RewriteRule "^/browse/MSONAR(.*)$" "http://jira.sonarsource.com/browse/MSONAR$1" [L]
+RewriteRule "^/browse/SONARSCTFS-(.*)$" "http://jira.sonarsource.com/browse/SONARTFVC-$1" [R=301,L]
```
Comment (dbolkensteyn) [inline includes/jira.codehaus.org.inc]: @hgomez-sonarsource I'm not sure this works as expected. `SONARSCTFS-4` should map to `SONARTFVC-11`

On Codehaus, we had 2 projects: SONARTFS and SONARSCTFS. All the tickets from SONARSCTFS got merged into the SONARTFS project - but this operation changed the ticket numbers. Jira was able to keep track of that.

Comment (dbolkensteyn) [inline includes/jira.codehaus.org.inc]: Also note that this affects less than 10 tickets in total, so I can even image to hardcode all the mappings in this file

Comment (hgomez-sonarsource) [inline includes/jira.codehaus.org.inc]: Well, in this case it would be fine to get translation mapping.

File: includes/jira.codehaus.org.inc [hunk 2]
```diff
@@ -7,8 +7,9 @@
 ##################################
 # Owned by: Olivier Gaudin       #
 ##################################
-RewriteRule "^/browse/SONAR(.*)$" "http://jira.sonarsource.com/browse/SONAR$1" [L]
-RewriteRule "^/browse/MSONAR(.*)$" "http://jira.sonarsource.com/browse/MSONAR$1" [L]
+#RewriteRule "^/browse/SONARSCTFS-(.*)$" "http://jira.sonarsource.com/browse/SONARTFVC-$1" [R=301,L]
```
Comment (dbolkensteyn) [inline includes/jira.codehaus.org.inc]: @hgomez-sonarsource Here are the mappings, there are just 4 of them in the end:
- SONARSCTFS-1 -> SONARTFVC-12
- SONARSCTFS-2 -> SONARTFVC-14
- SONARSCTFS-3 -> SONARTFVC-13
- SONARSCTFS-4 -> SONARTFVC-11

**Comment by bwalding** [comment]
Deployed.

**Comment by hgomez-sonarsource** [comment]
Thanks Ben

**Pull request merged**